Tuleap/RestClient/ArtifactParser.py

```diff
@@ -267,3 +270,14 @@ def __extract_links(self, item):
                             # Append only the artifacts that belong to the correct tracker (next tracker in the chain)!
                             if trck_id_tmp == self.__child_id:
                                 self.__reverse_links.append(id_tmp)
+
+    def __reset_list(self, items):
+        """
+        Clear list with python2.7 compatibility
+
+        :type items: list
+        """
+        try:
```

Comment:
Isn't there a more elegant way to check the python version? Would some of the [solutions mentioned here](https://stackoverflow.com/questions/1093322/how-do-i-check-what-version-of-python-is-running-my-script) be better than using exceptions?